tools/istio-iptables/pkg/cmd/root.go

```diff
@@ -44,8 +44,8 @@ var (
 	// InvalidDropByIptables is the flag to enable invalid drop iptables rule to drop the out of window packets
 	InvalidDropByIptables = env.Register("INVALID_DROP", false,
 		"If set to true, enable the invalid drop iptables rule, default false will cause iptables reset out of window packets")
-	DualStack = env.RegisterBoolVar("ISTIO_DUAL_STACK", false,
-		"If true, Istio will enable the Dual Stack feature.").Get()
+	DualStack = env.Register("ISTIO_DUAL_STACK", false,
```

Comment:
did we need to remove the Get() here? I think its for the test, will comment there the preferred way to set it